src/BTVNanoCommissioning/workflows/ttsemilep_valid_sf.py

```diff
@@ -58,7 +61,126 @@ def accumulator(self):
+            and hasattr(events.Jet, "btagDeepFlavB_lepb")
+            else None
+        )
+        (
+            print("ProbC is: ", events.Jet.btagDeepFlavC)
+            if hasattr(events.Jet, "btagDeepFlavC")
+            else None
+        )
+        (
+            print(
+                "ProbC calculated is: ",
+                events.Jet.btagDeepFlavCvL
+                / (1.0 - events.Jet.btagDeepFlavCvL)
+                * (events.Jet.btagDeepFlavG + events.Jet.btagDeepFlavUDS),
+            )
+            if hasattr(events.Jet, "btagDeepFlavCvL")
+            and hasattr(events.Jet, "btagDeepFlavUDS")
+            and hasattr(events.Jet, "btagDeepFlavG")
+            else None
+        )
+        (
+            print(
+                "ProbC calculated is : ",
+                events.Jet.btagDeepFlavCvB
+                / (1.0 - events.Jet.btagDeepFlavCvB)
+                * (events.Jet.btagDeepFlavB),
+            )
+            if hasattr(events.Jet, "btagDeepFlavCvB")
+            else None
+        )
+
         events = missing_branch(events)
+        (
+            print("ProbB present in dataset after corr")
+            if hasattr(events.Jet, "btagDeepFlavB")
+            else None
+        )
+        (
+            print("ProbC present in dataset after corr")
+            if hasattr(events.Jet, "btagDeepFlavC")
+            else None
+        )
+        (
+            print("ProbB is after corr: ", events.Jet.btagDeepFlavB)
+            if hasattr(events.Jet, "btagDeepFlavB")
+            else None
+        )
+        (
+            print(
+                "ProbB calculated is after corr: ",
+                events.Jet.btagDeepFlavB_b
+                + events.Jet.btagDeepFlavB_bb
+                + events.Jet.btagDeepFlavB_lepb,
+            )
+            if hasattr(events.Jet, "btagDeepFlavB_b")
+            and hasattr(events.Jet, "btagDeepFlavB_bb")
+            and hasattr(events.Jet, "btagDeepFlavB_lepb")
+            else None
+        )
+        (
+            print("ProbC is after corr: ", events.Jet.btagDeepFlavC)
+            if hasattr(events.Jet, "btagDeepFlavC")
+            else None
+        )
+        (
+            print(
+                "ProbC calculated is after corr: ",
+                events.Jet.btagDeepFlavCvL
+                / (1.0 - events.Jet.btagDeepFlavCvL)
+                * (events.Jet.btagDeepFlavG + events.Jet.btagDeepFlavUDS),
+            )
+            if hasattr(events.Jet, "btagDeepFlavCvL")
+            and hasattr(events.Jet, "btagDeepFlavUDS")
+            and hasattr(events.Jet, "btagDeepFlavG")
+            else None
+        )
+        (
+            print(
+                "ProbC calculated is after corr B: ",
+                events.Jet.btagDeepFlavCvB
+                / (1.0 - events.Jet.btagDeepFlavCvB)
+                * (events.Jet.btagDeepFlavB),
+            )
+            if hasattr(events.Jet, "btagDeepFlavCvB")
+            else None
+        )
```

Comment:
please remove all the print statements here

Reply:
Removed